### PR TITLE
fix lars dist support

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/Paddle/issues/13386

### CODE REVIEW

**Comment by typhoonzero** [comment]
@panyx0718 @qingqing01 Need review for API change, thank you!

File: paddle/fluid/API.spec [hunk 1]
```diff
@@ -353,7 +353,7 @@ paddle.fluid.nets.glu ArgSpec(args=['input', 'dim'], varargs=None, keywords=None
 paddle.fluid.nets.scaled_dot_product_attention ArgSpec(args=['queries', 'keys', 'values', 'num_heads', 'dropout_rate'], varargs=None, keywords=None, defaults=(1, 0.0))
 paddle.fluid.optimizer.SGDOptimizer.__init__ ArgSpec(args=['self', 'learning_rate'], varargs=None, keywords='kwargs', defaults=None)
 paddle.fluid.optimizer.SGDOptimizer.minimize ArgSpec(args=['self', 'loss', 'startup_program', 'parameter_list', 'no_grad_set'], varargs=None, keywords=None, defaults=(None, None, None))
-paddle.fluid.optimizer.MomentumOptimizer.__init__ ArgSpec(args=['self', 'learning_rate', 'momentum', 'use_nesterov'], varargs=None, keywords='kwargs', defaults=(False,))
+paddle.fluid.optimizer.MomentumOptimizer.__init__ ArgSpec(args=['self', 'learning_rate', 'momentum', 'use_nesterov', 'use_lars', 'lars_coeff', 'lars_weight_decay'], varargs=None, keywords='kwargs', defaults=(False, False, 0.001, 0.0005))
```
Comment (panyx0718) [inline paddle/fluid/API.spec]: Does this mean every optimizer needs to have these arguments to enable "lars"?

**Review by panyx0718 — COMMENTED** [review]
Let me think more about it. We should be more serious about the AP changes

File: paddle/fluid/API.spec (hunk 1 above)
Comment (typhoonzero) [reply paddle/fluid/API.spec]: No, the paper only describes implementation for SGD and momentum, and seems momentum is for most of the cases.

Comment (typhoonzero) [reply paddle/fluid/API.spec]: As discussed with @panyx0718 will put this implementation in another op.

**Pull request closed without merging**